src/parseOption.ts

```diff
@@ -1,5 +1,5 @@
 import {autoFormat, customFormat} from './formatFieldValue';
-import {FieldOption, Option, Scenegraph, SupplementedFieldOption, TooltipData} from './options';
+import { FieldOption, Option, Scenegraph, ScenegraphData, SupplementedFieldOption, TemporaryFieldOption, TooltipData } from './options';
```

Comment:
The space after the {  is inconsistent. We have rules in Vega lite's tslint to avoid this.